docs/hugo/content/guide/authentication/credential-format.md

```diff
@@ -1,35 +1,45 @@
 ---
-title: Authentication in Azure Service Operator v2
-linktitle: Authentication
+title: Credential Format
+linktitle: Credential Format
 ---
 
-Azure Service Operator supports four different styles of authentication today. Each of these options can be used either as a global credential applied to all resources created by the operator (as shown below), or as a per-resource or per-namespace credential as documented in [single-operator-multitenancy](../multitenant-deployment#single-operator-multitenancy-default).
-
-1. Service Principal using a Client Secret
-2. Service Principal using a Client Certificate
-3. [Azure-Workload-Identity](https://github.com/Azure/azure-workload-identity) authentication (OIDC + Managed Identity or Service Principal)
-4. [Deprecated] aad-pod-identity authentication (Managed Identity)
-
-
 ## Service Principal using a Client Secret
 
 ### Prerequisites
 1. An existing Azure Service Principal.
 
-To use Service Principal authentication with **client secret**, specify an `aso-controller-settings` secret with the `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET` keys set.
+To use Service Principal authentication with **client secret**, create a secret with the `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET` keys set.
 
 For more information about Service Principals, see [creating an Azure Service Principal using the Azure CLI](https://docs.microsoft.com/cli/azure/create-an-azure-service-principal-azure-cli#password-based-authentication).
 The `AZURE_CLIENT_ID` is sometimes also called the App ID. The `AZURE_CLIENT_SECRET` is the "password" returned by the command in the previously linked documentation.
 
-Use the following Bash script to set the environment variables for the `aso-controller-settings` secret:
+Use the following Bash script to set the environment variables for the service principal secret:
```

Comment:
It's not a script, it a set of commands (no `#!` at the start)

```suggestion
Use the following Bash commands to set the environment variables containing the service principal secret (customize with your values):
```

Reply:
Done